src/solver/infeasible-problem-analysis/constraint.cpp

```diff
@@ -75,67 +61,36 @@ class StringIsNotWellFormated: public std::runtime_error
     }
 };
 
-std::string StringBetweenAngleBrackets(const std::string& str)
+std::string StringBetweenAngleBrackets(const std::string& constraintName)
```

Comment:
this function is agnostic to its input and therefore the param should not be renamed;